docs/topics/kapt.md

```diff
@@ -167,6 +167,70 @@ If you run into any problems with caching for annotation processors, disable cac
 kapt.classloaders.cache.disableForProcessors=[annotation processors full names]
 ```
 
+### Measuring performance of annotation processors
+
+You can get a performance statistics on the annotation processors execution using the `-Kapt-show-processor-timings` plugin option. 
+An example output:
+
+```
+Kapt Annotation Processing performance report:
+com.example.processor.TestingProcessor: total: 133 ms, init: 36 ms, 2 round(s): 97 ms, 0 ms
+com.example.processor.AnotherProcessor: total: 100 ms, init: 6 ms, 1 round(s): 93 ms
+```
+
+You can dump this report into a file with the plugin option [`-Kapt-dump-processor-timings` (`org.jetbrains.kotlin.kapt3:dumpProcessorTimings`)](https://github.com/JetBrains/kotlin/pull/4280). 
+The following command will run kapt and dump the statistics to the file `ap-perf-report.file`:
+
+```
+kotlinc -cp $MY_CLASSPATH \
+-Xplugin=kotlin-annotation-processing-SNAPSHOT.jar -P \
+plugin:org.jetbrains.kotlin.kapt3:aptMode=stubsAndApt,\
+plugin:org.jetbrains.kotlin.kapt3:apclasspath=processor/build/libs/processor.jar,\
+plugin:org.jetbrains.kotlin.kapt3:dumpProcessorTimings=ap-perf-report.file \
+-Xplugin=$JAVA_HOME/lib/tools.jar \
+-d cli-tests/out \
+-no-jdk -no-reflect -no-stdlib -verbose \
+sample/src/main/
+```
+
+### Measuring number of files generated with annotation processors
+
+The `kotlin-kapt` Gradle plugin can report statistics on the number of generated files for each annotation processor.
```

Comment:
For tech writers: probably I need to start both these sections similarly? What do you think is better: "You can..." or "The `kotlin-kapt` Gradle plugin can report statistics..."?